tests/components/brunt/test_config_flow.py

```diff
@@ -0,0 +1,157 @@
+"""Test the Brunt config flow."""
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.brunt.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from tests.common import MockConfigEntry
+
+CONF = {CONF_USERNAME: "test-username", CONF_PASSWORD: "test-password"}
```

Comment:
`CONF` is a constant we us to prefix config keys in the constant name.
```suggestion
CONFIG = {CONF_USERNAME: "test-username", CONF_PASSWORD: "test-password"}
```